treeherder/log_parser/management/commands/crossreference_error_lines.py

```diff
@@ -96,20 +88,20 @@ def crossreference_error_lines(self, repository, job_guid, failure_lines, text_l
 
         # For each error in the text log summary artifact, try to match the next
```

Comment:
nit, but the comment here should be adjusted since there's no more artifact.  :)